bot/events/voiceChannelJoin.js

```diff
@@ -0,0 +1,19 @@
+const {EventListener} = require('yuuko');
+module.exports = new EventListener('voiceChannelJoin', (user, channel) => {
+	const voiceRoleId = '336954476315541515'; // TODO: put these in a file for role/channel ids
+	const radioRoleId = '710506067943227416';
+	const radioChannelId = '290594195687735298';
+
+	if (channel.id === radioChannelId) {
+		// remove the wrong role first, if the user somehow has it. If the user already has the role, do nothing
+		if (user.roles.includes(voiceRoleId)) user.removeRole(voiceRoleId, 'Radio Channel Joined');
```

Comment:
`Member#removeRole` returns a `Promise`, so it needs to be `await`ed and poissible rejections need to bne handled (either `try`/`catch` or tagged with `.catch(() => {})` - in this case we probably want an error message if the role can't be removed for some reason). Same goes for `addRole` below, as well as in the `voiceChannelLeave` and `voiceChannelSwitch` handlers..